Style2

margotTenenbaum opened the pull request:
Hi @GrumpyCat0709 and @Nick-Zabalza,

This PR covers styling improvements and the addition of the modal to my component. The service has been left in proxy-mode, as I believe at this point that allows the styling/modal to show up most like the real website. (Please follow the README if you would prefer to look at it in dev mode.) Please note: this service shows up very close to the photoGallery service at present (ie not enough margin). This will need to be addressed at the proxy level, and I feel that will be most efficiently handled once the third top-row component is also available.

Styling improvements: +/- button color change, improved font and line spacing, removal of grey border around small images, fix styling on quantity bar borders

One remaining: I was unable to substantially improve the half-star showing up slightly out of alignment. Something I will keep in mind if I have time to go back later this week.

Thank you!

Nick-Zabalza commented:
@margotTenenbaum 

Hi Elizabeth,

Looking better and better. Some issues though. I tried to do a pixel perfect coparison and noticed some discrepencies. Some of them are minor and others are major. I don't how close to pixel perfect we have to be and so i will separate the minor from the major, with the major highly recommended to be taken care of. Also, I may be mistaken regarding my measurements and so you can double check if you want.

Major
-5 additional pixels needed between stars and price
-20 additional pixels needed between price and gray line
-20 additional pixels needed between gray line and price gaurantee
- The price match gaurantee  needs to be moved to the right 5 pixels
- The qty bar needs to be moved to the right 5 pixels

Minor
- 1 too many pixels between the Title and By: brand
- 1 additional pixel needed between the By: brand and stars
- 1 additional pixel needed between the horizontal placement of the stars and (# of reviews)
- 1 too many pixels between the vertical placement of the stars and (# of reviews)
- 1 additional pixel needed between price gaurantee and qty bar
- The entire line with stars and (# of reviews) needs to be moved to the right 1 pixel

The modal also had some issues:
-Height of modal (no border) seems short by 48 pixels
-Width of modal (no border) seems short by 44 pixels
-Modal missing 2px border with a blue-family color
-Since the above CSS was off, I stopped there and so I don't know if the internal CSS is also off. Or positioning on page
-Modal does not alter the background color of MODAL_ATTACH_POINT and so the website behind the modal is not disabled. I suspect the background color should be white with 0.4 transparency but I did not confirm this.

Lastly, not all of your tests are still passing and I don't believe it has anything to do with different database data.

Good luck

margotTenenbaum commented:
@Nick-Zabalza Thank you! I addressed all of the major points and most of the minor ones. Also fixed the issue with the tests. I did some research on blurring the background when the modal is open, and I think that will need to wait for a future PR if I have the time to work on it after deploying.
